From 8d90e45cba42a45aa053140302a38f0049f35e1e Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Mon, 10 Feb 2025 15:57:35 +0800
Subject: [PATCH 325/360] deepin: arm64: config: disable ARCH_MA35 and NPCM

Clean the Embedded system config which is not used in desktop.

Signed-off-by: Wentao Guan <guanwentao@uniontech.com>
---
 .../configs/deepin_arm64_desktop_defconfig    | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index e3781ad54c49..c177bd641ec6 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -73,8 +73,6 @@ CONFIG_ARCH_NXP=y
 CONFIG_ARCH_LAYERSCAPE=y
 CONFIG_ARCH_MXC=y
 CONFIG_ARCH_S32=y
-CONFIG_ARCH_MA35=y
-CONFIG_ARCH_NPCM=y
 CONFIG_ARCH_PHYTIUM=y
 CONFIG_ARCH_QCOM=y
 CONFIG_ARCH_REALTEK=y
@@ -886,8 +884,6 @@ CONFIG_MTD_ONENAND_2X_PROGRAM=y
 CONFIG_MTD_RAW_NAND=m
 CONFIG_MTD_NAND_DENALI_PCI=m
 CONFIG_MTD_NAND_DENALI_DT=m
-CONFIG_MTD_NAND_OMAP2=m
-CONFIG_MTD_NAND_OMAP_BCH=y
 CONFIG_MTD_NAND_CAFE=m
 CONFIG_MTD_NAND_PHYTIUM_PCI=m
 CONFIG_MTD_NAND_PHYTIUM_PLAT=m
@@ -895,7 +891,6 @@ CONFIG_MTD_NAND_MARVELL=m
 CONFIG_MTD_NAND_BRCMNAND=m
 CONFIG_MTD_NAND_BRCMNAND_BCM63XX=m
 CONFIG_MTD_NAND_GPMI_NAND=m
-CONFIG_MTD_NAND_FSL_IFC=m
 CONFIG_MTD_NAND_MXC=m
 CONFIG_MTD_NAND_SUNXI=m
 CONFIG_MTD_NAND_HISI504=m
@@ -2044,7 +2039,6 @@ CONFIG_SERIAL_SPRD=m
 CONFIG_SERIAL_STM32=m
 CONFIG_SERIAL_MVEBU_UART=y
 CONFIG_SERIAL_OWL=m
-CONFIG_SERIAL_NUVOTON_MA35D1=m
 CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_MOXA_INTELLIO=m
 CONFIG_MOXA_SMARTIO=m
@@ -2062,7 +2056,6 @@ CONFIG_PPDEV=m
 CONFIG_VIRTIO_CONSOLE=y
 CONFIG_PHYTIUM_KCS_IPMI_BMC=m
 CONFIG_PHYTIUM_BT_IPMI_BMC=m
-CONFIG_NPCM7XX_KCS_IPMI_BMC=m
 CONFIG_IPMI_KCS_BMC_CDEV_IPMI=m
 CONFIG_IPMI_KCS_BMC_SERIO=m
 CONFIG_SSIF_IPMI_BMC=m
@@ -2137,7 +2130,6 @@ CONFIG_I2C_MESON=m
 CONFIG_I2C_MT65XX=m
 CONFIG_I2C_MV64XXX=m
 CONFIG_I2C_NOMADIK=m
-CONFIG_I2C_NPCM=m
 CONFIG_I2C_OCORES=m
 CONFIG_I2C_OMAP=m
 CONFIG_I2C_OWL=m
@@ -2215,9 +2207,6 @@ CONFIG_SPI_MICROCHIP_CORE=m
 CONFIG_SPI_MICROCHIP_CORE_QSPI=m
 CONFIG_SPI_MT65XX=m
 CONFIG_SPI_MTK_NOR=m
-CONFIG_SPI_WPCM_FIU=m
-CONFIG_SPI_NPCM_FIU=m
-CONFIG_SPI_NPCM_PSPI=m
 CONFIG_SPI_OC_TINY=m
 CONFIG_SPI_OMAP24XX=m
 CONFIG_SPI_ORION=y
@@ -3617,7 +3606,6 @@ CONFIG_EDAC_PHYTIUM=m
 CONFIG_EDAC_BLUEFIELD=m
 CONFIG_EDAC_DMC520=m
 CONFIG_EDAC_ZYNQMP=m
-CONFIG_EDAC_NPCM=m
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_ABB5ZES3=m
 CONFIG_RTC_DRV_ABX80X=m
@@ -3807,13 +3795,6 @@ CONFIG_EXTCON_MAX8997=m
 CONFIG_EXTCON_PALMAS=m
 CONFIG_EXTCON_RT8973A=m
 CONFIG_EXTCON_SM5502=m
-CONFIG_ARM_PL172_MPMC=m
-CONFIG_BRCMSTB_DPFE=m
-CONFIG_BRCMSTB_MEMC=m
-CONFIG_OMAP_GPMC=m
-CONFIG_FSL_IFC=y
-CONFIG_RENESAS_RPCIF=m
-CONFIG_STM32_FMC2_EBI=m
 CONFIG_IIO=m
 CONFIG_IIO_BUFFER_CB=m
 CONFIG_IIO_BUFFER_DMAENGINE=m
-- 
2.20.1

